### DISCUSSION TITLE
Verschlüsselungsfrage

### DISCUSSION THREAD

**Discussion by btesteracc** [lucaapp/security-concept#21] [Q&A]
Hallo!
Zunächst mal tolles Produkt. Einen Punkt habe ich aber nicht verstanden. Ihr schreibt, dass die Kontaktdaten nur durch das zuständige Gesundheitsamt eingesehen (entschlüsselt) werden können. Ich habe die Doku so verstanden, dass die Daten mit einem täglich wechselnden öffentlichen Schlüssel des Gesundheitsamtes verschlüsselt werden. Wenn mein Gesundheitsamt jetzt aber Lucas noch nicht nutzt, wie werden dann meine Kontaktdaten verschlüsselt, da sie ja auch einem Veranstalter oder Betreiber gegenüber nur verschlüsselt bereitgestellt werden. Wo kommen die Tagesschlüssel her mit denen die Daten verschlüsselt werden? Oder habe ich jetzt irgendeinen Denkfehler gemacht?

**Comment by reneme** [comment]
Vielen Dank für dein Interesse an Luca! Bitte entschuldige die etwas lange Antwortzeit. In unseren [FAQs](https://www.luca-app.de/faq/#ac_6955_collapse15) erklären wir dies unter "Ich will mein Restaurant/ Betrieb/Firma in luca Locations anlegen, aber leider kann ich nicht auf das System zugreifen, warum?":

> luca für Betreiber ist überall da verfügbar, wo Gesundheitsämter an das luca System angeschlossen sind. [...] Sobald luca auch an deinem Standort verfügbar ist, kannst du eine Location anlegen. Leider ist es nicht möglich, dass du deine Locations anlegst, wenn dein zuständiges Gesundheitsamt nicht ausgestattet ist.

Du kannst dir Luca natürlich unabhängig davon installieren, aber richtig nutzen kannst du die App leider erst, wenn das Gesundheitsamt in deiner Region auch angebunden ist.
